migration/ubuntu/20.04-to-22.04-openstack.md

```diff
@@ -0,0 +1,256 @@
+# Migrate to Ubuntu 24.04 on Openstack
+
+> [!IMPORTANT]
+> This migration assumes that the environment is running on an Openstack environment.
+> This mainly applies to the terraform details, so you should be able to follow along with the Kubespray steps even if you're not running on an Openstack environment.
```

Comment:
It "mainly" applies to the terraform details, but do we know what else it applies to? It seems like it would be good to test doing it on non-openstack environments as well (doesn't have to be as part of this task).

Reply:
Yea, that might be true. Maybe we should do it on our long running clusters? That would include upcloud atleast if SME have time to do it